---
title: 'Ecophysiological Features Shape the Distribution of Prophages and CRISPR in Sulfate Reducing Prokaryotes.'
authors:
  - admin
  - Alejandra Arancibia
  - Leonardo Badilla
  - Jonathan Acosta
  - Gabriela Arancibia
  - Rodrigo Escar
  - Gustavo Ferrada
  - Michael Seeger
    
#author_notes:
#  - 'Equal contribution'
#  - 'Equal contribution'
date: '2021-05-01T00:00:00Z'
doi: 'https://doi.org/10.3390/microorganisms9050931'

# Schedule page publish date (NOT publication's date).
publishDate: '2021-05-01T00:00:00Z'

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ['2']

# Publication name and optional abbreviated publication name.
publication: '*Microorganisms, 9*(5)'
publication_short: ''

abstract: Sulfate reducing prokaryotes (SRP) are a phylogenetically and physiologically diverse group of microorganisms that use sulfate as an electron acceptor. SRP have long been recognized as key players of the carbon and sulfur cycles, and more recently, they have been identified to play a relevant role as part of syntrophic and symbiotic relations and the human microbiome. Despite their environmental relevance, there is a poor understanding about the prevalence of prophages and CRISPR arrays and how their distribution and dynamic affect the ecological role of SRP. We addressed this question by analyzing the results of a comprehensive survey of prophages and CRISPR in a total of 91 genomes of SRP with several genotypic, phenotypic, and physiological traits, including genome size, cell volume, minimum doubling time, cell wall, and habitat, among others. Our analysis discovered 81 prophages in 51 strains, representing the 56% of the total evaluated strains. Prophages are non-uniformly distributed across the SRP phylogeny, where prophage-rich lineages belonged to Desulfovibrionaceae and Peptococcaceae. Furthermore, our study found 160 CRISPR arrays in 71 SRP, which is more abundant and widely spread than previously expected. Although there is no correlation between presence and abundance of prophages and CRISPR arrays at the strain level, our analysis showed that there is a directly proportional relation between cellular volumes and number of prophages per cell. This result suggests that there is an additional selective pressure for strains with smaller cells to get rid of foreign DNA, such as prophages, but not CRISPR, due to less availability of cellular resources. Analysis of the prophage genes encoding viral structural proteins reported that 44% of SRP prophages are classified as Myoviridae, and comparative analysis showed high level of homology, but not synteny, among prophages belonging to the Family Desulfovibrionaceae. We further recovered viral-like particles and structures that resemble outer membrane vesicles from D. vulgaris str. Hildenborough. The results of this study improved the current understanding of dynamic interactions between prophages and CRISPR with their hosts in both cultured and hitherto-uncultured SRP strains, and how their distribution affects the microbial community dynamics in several sulfidogenic natural and engineered environments.

# Summary. An optional shortened abstract.
# summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.

tags:
  - profagos
  - bacterias sulfato reductoras
  - virus
featured: true

# links:
# - name: ""
#   url: ""
url_pdf: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8146710/pdf/microorganisms-09-00931.pdf
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
image:
  caption: 'Image credit: [**Unsplash**](https://unsplash.com/photos/jdD8gXaTZsc)'
  focal_point: ''
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides:

---

{{% callout note %}}
Click the _Cite_ button above to demo the feature to enable visitors to import publication metadata into their reference management software.
{{% /callout %}}

Supplementary notes can be added here, including [code and math](https://wowchemy.com/docs/content/writing-markdown-latex/).

